src/commands/airdrop/index/processor.ts

```diff
@@ -271,11 +272,16 @@ export async function handleAirdrop(
   if (all) payload.amount = balance
 
   const tokenEmoji = getEmoji(payload.token)
-  const usdAmount = (usdBalance / balance) * payload.amount
+  let usdAmount = await convertToUsdValue(payload.amount, payload.token)
```

Comment:
pls provide screenshot for this bug fix
i dont want to repeat myself every time

Reply:
<img width="547" alt="Screen Shot 2023-04-04 at 2 12 37 PM" src="https://user-images.githubusercontent.com/39359294/229715887-97da46c7-9fc6-4dba-aadf-324b535d7835.png">

Reply:
<img width="558" alt="Screen Shot 2023-04-04 at 2 15 56 PM" src="https://user-images.githubusercontent.com/39359294/229716649-4b3b4f69-00f3-421d-95ec-fed8a7ab4234.png">